.github/workflows/todo-app-pipeline-test.yml

```
@@ -0,0 +1,22 @@
+name: Continuous Integration
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build the docker image
+      run: docker build --target test --tag todo-app:test .
+
+    - name: Run the docker test image
+      run:  docker run -p 5000:5000 --env-file .env.test todo-app:test
```

Not that it causes any harm, but note that nothing external is talking to this container, so no need to publish any ports
```suggestion
      run:  docker run --env-file .env.test todo-app:test
```